packages/components/organisms/f-checkout/src/components/Checkout.vue

```diff
@@ -395,6 +396,17 @@ export default {
             return invalidFieldCount === 1 ?
                 this.$t('errorMessages.singleFieldError') :
                 this.$t('errorMessages.multipleFieldErrors', { errorCount: invalidFieldCount });
+        },
+
+        formattedMobileNumberForScreenReader () {
+            const usesAreaCode = this.customer.mobileNumber.startsWith('+');
```

Comment:
`hasAreaCode`

Reply:
Following the above comment this code `hasBeenDeleted` 😅